pandas/core/frame.py

```diff
@@ -3924,30 +3925,46 @@ def rename(self, *args, **kwargs):
 
         Parameters
         ----------
-        mapper, index, columns : dict-like or function, optional
-            dict-like or functions transformations to apply to
+        mapper : dict-like or function
+            Dict-like or functions transformations to apply to
             that axis' values. Use either ``mapper`` and ``axis`` to
             specify the axis to target with ``mapper``, or ``index`` and
             ``columns``.
-        axis : int or str, optional
+        index : dict-like or function
+            Alternative to specifying axis (``mapper, axis=0``
+            is equivalent to ``index=mapper``).
+        columns : dict-like or function
+            Alternative to specifying axis (``mapper, axis=1``
+            is equivalent to ``columns=mapper``).
+        axis : int or str
             Axis to target with ``mapper``. Can be either the axis name
             ('index', 'columns') or number (0, 1). The default is 'index'.
-        copy : boolean, default True
-            Also copy underlying data
-        inplace : boolean, default False
+        copy : bool, default True
+            Also copy underlying data.
+        inplace : bool, default False
             Whether to return a new DataFrame. If True then value of copy is
             ignored.
         level : int or level name, default None
             In case of a MultiIndex, only rename labels in the specified
             level.
+        errors : {'ignore', 'raise'}, default 'ignore'
+            If 'ignore', suppress error and existing labels are renamed.
```

Comment:
I think this could be clearer. Perhaps

    If 'raise', raise a `KeyError` when a dict-like `mapper`, `index`, or `columns` contains
    labels that are not present in the Index being transformed.

Reply:
Do you think it's best to include both the ignore and raise description?

Reply:
Yeah. Maybe append a "If 'ignore', extra keys will be ignored"? Something along those lines?

I just want to avoid the impression that passing "extra" keys is somehow an mis-use of `.rename`.